2023-09-03 10:48:17,816 ERROR: Object reference not set to an instance of an object.
   at AuthenticationJWT.Controllers.LoginController.<>c__DisplayClass7_0.<IsEmailInUse>b__0() in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Controllers\LoginController.cs:line 119
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at AuthenticationJWT.Controllers.LoginController.<IsEmailInUse>d__7.MoveNext() in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Controllers\LoginController.cs:line 119
2023-09-03 10:53:30,015 ERROR: Object reference not set to an instance of an object.
   at AuthenticationJWT.Controllers.LoginController.<>c__DisplayClass7_0.<IsEmailInUse>b__0() in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Controllers\LoginController.cs:line 119
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at AuthenticationJWT.Controllers.LoginController.<IsEmailInUse>d__7.MoveNext() in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Controllers\LoginController.cs:line 119
2023-09-03 10:57:52,228 ERROR: Object reference not set to an instance of an object.
   at AuthenticationJWT.Controllers.LoginController.<>c__DisplayClass7_0.<IsEmailInUse>b__0() in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Controllers\LoginController.cs:line 119
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at AuthenticationJWT.Controllers.LoginController.<IsEmailInUse>d__7.MoveNext() in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Controllers\LoginController.cs:line 119
2023-09-03 11:00:37,619 ERROR: Object reference not set to an instance of an object.
   at AuthenticationJWT.Controllers.LoginController.<>c__DisplayClass7_0.<IsEmailInUse>b__0() in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Controllers\LoginController.cs:line 119
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at AuthenticationJWT.Controllers.LoginController.<IsEmailInUse>d__7.MoveNext() in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Controllers\LoginController.cs:line 119
2023-09-07 18:45:02,180 ERROR: IDX10223: Lifetime validation failed. The token is expired. ValidTo (UTC): '07-09-2023 13:09:40', Current time (UTC): '07-09-2023 13:14:44'.
   at Microsoft.IdentityModel.Tokens.Validators.ValidateLifetime(Nullable`1 notBefore, Nullable`1 expires, SecurityToken securityToken, TokenValidationParameters validationParameters)
   at System.IdentityModel.Tokens.Jwt.JwtSecurityTokenHandler.ValidateLifetime(Nullable`1 notBefore, Nullable`1 expires, JwtSecurityToken jwtToken, TokenValidationParameters validationParameters)
   at System.IdentityModel.Tokens.Jwt.JwtSecurityTokenHandler.ValidateTokenPayload(JwtSecurityToken jwtToken, TokenValidationParameters validationParameters, BaseConfiguration configuration)
   at System.IdentityModel.Tokens.Jwt.JwtSecurityTokenHandler.ValidateJWS(String token, TokenValidationParameters validationParameters, BaseConfiguration currentConfiguration, SecurityToken& signatureValidatedToken, ExceptionDispatchInfo& exceptionThrown)
--- End of stack trace from previous location where exception was thrown ---
   at System.IdentityModel.Tokens.Jwt.JwtSecurityTokenHandler.ValidateToken(String token, JwtSecurityToken outerToken, TokenValidationParameters validationParameters, SecurityToken& signatureValidatedToken)
   at System.IdentityModel.Tokens.Jwt.JwtSecurityTokenHandler.ValidateToken(String token, TokenValidationParameters validationParameters, SecurityToken& validatedToken)
   at AuthenticationJWT.Utils.JwtHelper.GetClaimsPrincipalFromToken(String token) in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Utils\JwtHelper.cs:line 38
   at AuthenticationJWT.Controllers.HomeController.<>c__DisplayClass4_0.<Index>b__0() in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Controllers\HomeController.cs:line 39
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at AuthenticationJWT.Controllers.HomeController.<Index>d__4.MoveNext() in F:\Project\MyProject\AuthenticationJWT\AuthenticationJWT\Controllers\HomeController.cs:line 39
